### PR TITLE
39 make it possible to run on devices with 6 display

### PR DESCRIPTION
**Pull request by gscdev**
Depends on #49

### CODE REVIEW

**Comment by nya-elimuai** [comment]
6.3" (API 27):

device-2020-06-28-183714

device-2020-06-28-184027

device-2020-06-28-183921

device-2020-06-28-183949

**Comment by nya-elimuai** [comment]
7" (API 24):

device-2020-06-28-184358

device-2020-06-28-184408

device-2020-06-28-184435

device-2020-06-28-184441

**Comment by nya-elimuai** [comment]
7.02" (API 25):

device-2020-06-28-190016

device-2020-06-28-190101

device-2020-06-28-190116

device-2020-06-28-190124

**Comment by nya-elimuai** [comment]
8.4" (API 24):

device-2020-06-28-190853

device-2020-06-28-190904

device-2020-06-28-190934

device-2020-06-28-190942

**Comment by gscdev** [comment]
@nya-elimuai I can't reproduce the error with an emulator, but I added a new dimen.xml

**Comment by gscdev** [comment]
I've added ConstraintLayout and use percentages

File: app/src/main/res/layout/item_egra_oral_vocabulary.xml [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/egraOralVocabularyContainer"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:padding="8dp"
-    android:gravity="center">
+    android:maxHeight="@dimen/spaceship_height"
+    tools:layout_width="@dimen/spaceship_width"
```
Comment (nya-elimuai) [inline app/src/main/res/layout/item_egra_oral_vocabulary.xml]: Should `tools:layout_width` match the value of `android:layout_width` (for better consistency between the layout in the Android Studio preview and when running on a physical device?

**Comment by nya-elimuai** [comment]
@gscdev Ready to be merged. Thank you :-)

File: app/src/main/res/layout/item_egra_oral_vocabulary.xml (hunk 1 above)
Comment (gscdev) [reply app/src/main/res/layout/item_egra_oral_vocabulary.xml]: `item_*` layout doesn't have a defined size, its size is determined by `fragment_home_screen`. With `tools:layout_width="wrap_content"` images aren't visible in preview

**Pull request merged**